src/actions/search.js

```diff
@@ -0,0 +1,16 @@
+const SEARCH_INPUT = "SEARCH_INPUT";
+const SEARCH_FAVORITE = "SEARCH_FAVORITE";
+
+const setSearchInput = (query = "") => ({ type: SEARCH_INPUT, query });
```

Comment:
Remember that the related `data` of an action should be contained in a `payload` attribute.